Leaflet.Graticule.js

```diff
@@ -193,13 +193,30 @@ L.LatLngGraticule = L.Layer.extend({
         }
 
         // todo: format type of float
+        var dir = '';
+
         if (lat < 0) {
-            return '' + (lat*-1) + 'S';
+            lat = lat*-1;
+            dir = 'S';
         }
         else if (lat > 0) {
-            return '' + lat + 'N';
+            dir = 'N';
         }
-        return '' + lat;
+
+        var zoom = this._map.getZoom();
+        lat = '' + lat;
+
+        if(lng.indexOf(".") != -1) {
+          if(zoom >= 9 && zoom < 13)
```

Comment:
No, actually i think you could just create a new variable there, calculated from the `lat` or `lon` -- and then use it a few lines later.